Find a functionally equivalent code:android.graphics.Canvas.saveLayerAlpha:COMMENT
Method Modifier: public      
Comment:/**
 * Helper for saveLayerAlpha() that takes 4 values instead of a RectF.
 */

@@ -1,3 +1,4 @@
 {
-    return native_saveLayerAlpha(mNativeCanvas, left, top, right, bottom, alpha, saveFlags);
+    alpha = Math.min(255, Math.max(0, alpha));
+    return native_saveLayerAlpha(mNativeCanvasWrapper, left, top, right, bottom, alpha, saveFlags);
 }

